stores/nvidia.py

```diff
@@ -222,6 +222,7 @@ def __init__(self, gpu, locale="en_us"):
                     self.nvidia_password = self.config["NVIDIA_PASSWORD"]
                     self.auto_buy_enabled = self.config["FULL_AUTOBUY"]
                     self.cvv = self.config.get("CVV")
+                    self.interval=int(self.config.get("INTERVAL"))
```

Comment:
This will cause errors for existing users until they update the config

Reply:
Oh yeah, I see what you mean, didn't think of that.  I'll try to create a default value of 5, and it's getting overridden with this value if it's get's added.

Reply:
Change it to `self.interval=int(self.config.get("INTERVAL", 3))`